include/state-observation/dynamics-estimators/kinetics-observer.hpp

```diff
@@ -867,7 +957,9 @@ class STATE_OBSERVATION_DLLAPI KineticsObserver : protected DynamicalSystemFunct
   {
     virtual ~IMU() {}
     IMU() : Sensor(sizeIMUSignal) {}
-    Kinematics kinematics;
+    int num;
```

Comment:
I added this variable because MCKineticsObserver also uses a list of IMUs, so I wanted to make sure that the indexes of both IMUs match and to make it clearer